include/m_lib.h

```diff
@@ -3,71 +3,81 @@
 
 #include "ultra64.h"
 #include "z64math.h"
-#include "unk.h"
+#include "../src/overlays/gamestates/ovl_play/m_play.h"
+#include "color.h"
+#include "sys_math.h"
 
-struct Game_Play;
 
-void mem_copy(void* dst, void* src, u32 size);
-s32 mem_clear(void* ptr, size_t size, u8 value);
-// void mem_cmp();
+typedef struct {
+    u32 cont:   1;
+    u32 type:   4;
+    u32 offset: 11;
+    s32 value:  16;
+} ValueSet; 
+
+extern void (*D_80107B00_jp[])(u8*,  ValueSet*); 
```

Comment:
This is an array of function pointers, right?
I think it would be a bit clearer if the function pointer was typedef'd and this was declared as an array of said typedef